src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/DataSetsController.cs

```diff
@@ -0,0 +1,46 @@
+﻿#nullable enable
+using System.Net.Mime;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Cache;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Cache;
+using GovUk.Education.ExploreEducationStatistics.Content.Requests;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using static GovUk.Education.ExploreEducationStatistics.Common.Cache.CronSchedules;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Controllers;
+
+[ApiController]
+[Route("api/data-sets")]
+[Produces(MediaTypeNames.Application.Json)]
+public class DataSetsController : ControllerBase
+{
+    private readonly IDataSetService _dataSetService;
+
+    public DataSetsController(IDataSetService dataSetService)
+    {
+        _dataSetService = dataSetService;
+    }
+
+    [HttpGet]
+    [MemoryCache(typeof(ListDataSetsCacheKey), durationInSeconds: 10, expiryScheduleCron: HalfHourlyExpirySchedule)]
```

Comment:
Depending on whether or not we want to include UI tests with this, we may want to have the `durationInSeconds` configurable per environment, so that it expires much faster in local development.  Not super important though for this PR.

Reply:
Discussed this with @duncan-at-hiveit and it should already apply to all instances using `MemoryCacheAttribute` if it has been overridden in the environment.

Reply:
:+1: